src/main/webapp/wise5/services/studentDataService.ts

```diff
@@ -457,8 +457,12 @@ export class StudentDataService extends DataService {
     return true;
   }
 
-  evaluateCriteria(criteria) {
-    return this.criteriaFunctionNameToFunction[criteria.name](criteria);
+  evaluateCriteria(criteria: any): boolean {
+    const criteriaFunction = this.criteriaFunctionNameToFunction[criteria.name];
+    if (criteriaFunction == null) {
+      return true;
+    }
+    return criteriaFunction(criteria);
```

Comment:
If the issue occurs when criteria.name is empty string (""), then should we change this check to
```suggestion
    if (criteria.name === "") {
      return true;
    }
    return this.criteriaFunctionNameToFunction[criteria.name](criteria);
```